client/src/components/categorical/index.tsx

```diff
@@ -27,22 +30,10 @@ class Categories extends React.Component<{}, State> {
     super(props);
     this.state = {
       createAnnoModeActive: false,
-      newCategoryText: "",
```

Comment:
all of these lines were removed because eslint complained about them. these states are not used anywhere

Reply:
wait never mind i think they are used in other places. lol.